docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.3'
 services:
   mysql:
-    image: mysql:5.7.38
+    image: mysql:8.0
```

Comment:
the last version of mysql docker image is 8.0.33:
https://hub.docker.com/_/mysql/tags

Reply:
I change the mysql docker image to 8.0.33 and is working

Reply:
And I add the gradle plugin configuration, I analyzed the code then i´d need add the buildscrpit block, beacuse we had yhe configurantion in plugun, like is wrote in quick stat, then i just added the apply docker compose.
Its ok or it not ok?

Reply:
I added the buildscript block!